sklearn/utils/estimator_checks.py

```diff
@@ -812,7 +816,7 @@ def check_estimators_pickle(name, Estimator):
                       random_state=0, n_features=2, cluster_std=0.1)
 
     # some estimators can't do features less than 0
```

Comment:
change to "some estimators require nonnegative or strictly positive features" and copy that elsewhere